ops/services/app_functions/report_stream_batched_publisher/functions/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
```

Comment:
Curious if this version here relates at all to the azure version in the package.json ? 

`version
This value indicates the schema version of host.json. The version string "version": "2.0" is required for a function app that targets the v2 runtime, or a later version. There are no host.json schema changes between v2 and v3.`

Although it states there are no differences between v2 and v3 it seems it would be best practice ? I could be off here thouse

Reply:
Ah it appears only options are version 2 and version 1 https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json

Reply:
Ah, thanks for checking!